src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -12024,13 +12024,343 @@ public static class Extension
         }
 
         [Fact]
-        public async Task NoSymbolCompletionsInEnumBaseList()
+        public async Task EnumBaseList1()
         {
             var source = "enum E : $$";
 
+            await VerifyItemExistsAsync(source, "System");
+
+            // Not accessible in the given context
+            await VerifyItemIsAbsentAsync(source, "Byte");
+            await VerifyItemIsAbsentAsync(source, "SByte");
+            await VerifyItemIsAbsentAsync(source, "Int16");
+            await VerifyItemIsAbsentAsync(source, "UInt16");
+            await VerifyItemIsAbsentAsync(source, "Int32");
+            await VerifyItemIsAbsentAsync(source, "UInt32");
+            await VerifyItemIsAbsentAsync(source, "Int64");
+            await VerifyItemIsAbsentAsync(source, "UInt64");
```

Comment:
consider just defining these as the set of names we would expect, then use a theory on many of these.